source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -555,11 +581,17 @@ export default class NetworkStatusStore extends Store {
     }
   };
 
+  @action _resetNTPRechecks = () => {
+    clearTimeout(this.NTPRecheckInterval);
```

Comment:
@daniloprates please add the following line after this one:
```
this.NTPRecheckInterval = null;
```
...to be in line with the other clear intervals: https://github.com/input-output-hk/daedalus/blob/a973d8bbfc7d045d1bb5f07ba18db5cd8f53c18c/source/renderer/app/stores/NetworkStatusStore.js#L634